src/main/java/by/onliner/page/components/Menu.java

```diff
@@ -0,0 +1,42 @@
+package by.onliner.page.components;
+
+import by.onliner.page.AbstractComponent;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class Menu extends AbstractComponent {
+
+    ShoppingCartPopup shoppingCartPopup;
```

Comment:
1) please use modifier
2) is popup part of the menu?

Reply:
1. used
2. not

Reply:
why do you need it here then?